kore/src/Kore/Internal/SideCondition.hs

```diff
@@ -144,18 +170,18 @@ instance InternalVariable variable => SQL.Column (SideCondition variable) where
     toColumn = SQL.toColumn . Pretty.renderText . Pretty.layoutOneLine . pretty
 
 instance TopBottom (SideCondition variable) where
```

Comment:
Why do we need this instance? It doesn't seem right to have one, to me, because we always assume the `SideCondition` is `\top`.